snuba/snuba_migrations/profile_chunks/0001_create_profile_chunks_table.py

```diff
@@ -0,0 +1,85 @@
+from typing import List, Sequence
+
+from snuba.clickhouse.columns import UUID, Column, DateTime64, UInt
+from snuba.clusters.storage_sets import StorageSetKey
+from snuba.migrations import migration, operations, table_engines
+from snuba.migrations.columns import MigrationModifiers as Modifiers
+from snuba.migrations.operations import OperationTarget, SqlOperation
+
+storage_set = StorageSetKey.PROFILE_CHUNKS
```

Comment:
Are you putting this on the profiles cluster or a new one? If you're using the profiles one, you can use the PROFILES storage_set_key

Reply:
Is it actually true?
```
snuba.migrations.groups.DuplicateStorageSetFoundInGroup: The storage_set=StorageSetKey.PROFILES is present in more than one MigrationGroup. Each storage_set should only reference one migration group.
```

Reply:
I think it's best to keep them separate, these are 2 different features for 2 different products even if they share the same cluster and part of the name.